src/frontend/src/meta_client.rs

```diff
@@ -130,6 +130,9 @@ pub trait FrontendMetaClient: Send + Sync {
     async fn list_rate_limits(&self) -> Result<Vec<RateLimitInfo>>;
 
     async fn get_meta_store_endpoint(&self) -> Result<String>;
+
+    async fn alter_sink_config(&self, sink_id: u32, config: BTreeMap<String, String>)
```

Comment:
please make the interface for more general use.

Reply:
I think the answer is yes in this case. 
The frontend always pass the normal props in BTreeMap/HashMap and the secrets in SecretRef. 
We are not altering secret here, just making sure referencing an existing secret can work. 

```
alter sink ... connector with ( some_props = secret <secret name>, ... );
```